setup.cfg

```diff
@@ -50,6 +51,7 @@ exclude=
     build/*
     docs/*
     venv*/*
+    local/*
```

Comment:
Where does this directory come from? I don't remember any tool that creates a "local" directory.

Reply:
It's entirely convention - and possibly entirely a personal one. I've found it's helpful to have a place in any given repo to put things that are related to the project, but *aren't* appropriate for version control - test projects, error logs for work in progress, temporary diff files etc.